CEN4910C_Project/main.py

```diff
@@ -347,13 +358,26 @@ def Neigh_names():
     },
     id = "bodyBackgroundColor"
     ),
+
+    
 ])
 
 container_2 = html.Div([

```

Comment:
This layout of the second page reverts the updated, simplified layout from the Flexing branch changes; consider revision for simplification and a more flexible layout (particularly with different sized screens or windows)

Reply:
Changed to Flexbox styling.